src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -544,6 +534,26 @@ export default class RadarTargetModel {
         return [true, 'ADJUST HALO'];
     }
 
+    /**
+     * Set default value of the scratchpad
+     *
+     * @for RadarTargetModel
+     * @method setDefaultScratchpad
+     * @private
+     * @chainable
+     */
+    setDefaultScratchpad() {
```

Comment:
This is actually a _public_ method, meaning it can be accessed from OUTSIDE the RadarTargetModel class (which we do-- the ScopeModel calls RadarTargetModel.setDefaultScratchpad). A private method would be one that is only allowed to be accessed from within the RadarTargetModel class. If no other classes _need_ to use a given method, it should be made private (because bugs).

Also, all private methods should begin with an underscore. Since this doesn't have one, that's how we know whether it's public/private at a glance when we go to use it. So if you ever find yourself in a situation where you're doing like `this.someObject._doSomething()`, you _know_ the designer of `someObject` has a reason why you probably shouldn't do that.

tldr: remove `@private`

Reply:
OK, good to know. That was leftover from when I copy-pasted the `_initializeScratchpad()` function and I didn't make that distinction with underscores and private functions. Pushing correction now.

Reply:
As I look back at this, I'm also removing `@chainable` since per our conversation that no longer applies and adding `@return {array} [success of operation, system's response]`. since that was missing